streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -68,20 +77,17 @@ FhirConverter createFhirConverter(CamelContext camelContext) throws IOException,
 	}
 	
 	private String getDebeziumConfig() throws IOException {
-		GeneralConfiguration gc = new GeneralConfiguration();
-		GeneralConfiguration generalConfiguration = gc.getFhirDebeziumConfigPath(params.fhirDebeziumConfigPath);
-		return "debezium-mysql:" + generalConfiguration.getDebeziumConfigurations().get("databaseHostName") + "?"
-		        + "databaseHostname=" + generalConfiguration.getDebeziumConfigurations().get("databaseHostName")
-		        + "&databaseServerId=" + generalConfiguration.getDebeziumConfigurations().get("databaseServerId")
-		        + "&databasePort=" + generalConfiguration.getDebeziumConfigurations().get("databasePort") + "&databaseUser="
-		        + generalConfiguration.getDebeziumConfigurations().get("databaseUser") + "&databasePassword="
-		        + generalConfiguration.getDebeziumConfigurations().get("databasePassword") + "&databaseServerName="
-		        + generalConfiguration.getDebeziumConfigurations().get("databaseName") + "&databaseWhitelist="
-		        + generalConfiguration.getDebeziumConfigurations().get("databaseSchema")
-		        + "org.apache.kafka.connect.storage.FileOffsetBackingStore" + "&offsetStorageFileName="
-		        + generalConfiguration.getDebeziumConfigurations().get("databaseOffsetStorage")
-		        + "&databaseHistoryFileFilename=" + generalConfiguration.getDebeziumConfigurations().get("databaseHistory")
-		        + "&snapshotMode=" + generalConfiguration.getDebeziumConfigurations().get("snapshotMode");
+		this.generalConfiguration = getFhirDebeziumConfigPath(params.fhirDebeziumConfigPath);
+		LinkedHashMap<String, String> debeziumConfigs = generalConfiguration.getDebeziumConfigurations();
+		return "debezium-mysql:" + debeziumConfigs.get("databaseHostName") + "?" + "databaseHostname="
+		        + debeziumConfigs.get("databaseHostName") + "&databaseServerId=" + debeziumConfigs.get("databaseServerId")
+		        + "&databasePort=" + debeziumConfigs.get("databasePort") + "&databaseUser="
+		        + debeziumConfigs.get("databaseUser") + "&databasePassword=" + debeziumConfigs.get("databasePassword")
+		        + "&databaseServerName=" + debeziumConfigs.get("databaseName") + "&databaseWhitelist="
+		        + debeziumConfigs.get("databaseSchema")
+		        + "&offsetStorage=org.apache.kafka.connect.storage.FileOffsetBackingStore" + "&offsetStorageFileName="
+		        + debeziumConfigs.get("databaseOffsetStorage") + "&databaseHistoryFileFilename="
+		        + debeziumConfigs.get("databaseHistory") + "&snapshotMode=" + debeziumConfigs.get("snapshotMode");
```

Comment:
@bashir2  see [here](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/master/streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java#L81)  ,
its supposed to be `&offsetStorage=org.apache.kafka.connect.storage.FileOffsetBackingStore`  not `org.apache.kafka.connect.storage.FileOffsetBackingStore`  otherwise it was being taken as part of the value for  `&databaseWhitelist=`

Reply:
Oh, I see, we should enable our streaming e2e soon :-)
Thanks for fixing this.